### DISCUSSION TITLE
Editing posts in the frontend. Selection options (title, author, category, etc.) are missing.

### DISCUSSION THREAD

**Discussion by WM-Loose** [joomla/joomla-cms#47065] [2. Cold Features]
Editors who only work in the frontend should also be able to adjust the options without having to go to the backend.

**Comment by chmst** [comment]
You want that editors can change the article options also in frontend. 
Category and status are available in the publishing tab.

Indeed: 
We have a param "show_article_options" in com_content, but this param is not used in frontend.

**Comment by WM-Loose** [comment]
addendum. Should be like that. In the frontend-article.xml the "attribs" part has been omitted, which also includes the "options". This is the article.xml for the backend https://forum.joomla.de/thread/16414-beitr%C3%A4ge-titel-verbergen-wird-ungewollt-beim-speichern-im-frontend-ge%C3%A4ndert/?postID=112252#post112252

**Comment by ChristineWk** [comment]
See here please: #38508
